src/components/LocationPage/NewLocationPageHeader.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { Fragment } from 'react';
+import CheckIcon from '@material-ui/icons/Check';
+import SummaryStats from 'components/SummaryStats/SummaryStats';
+import {
+  ColoredHeaderBanner,
+  HeaderContainer,
+  HeaderTitle,
+  HeaderSection,
+  HeaderSubCopy,
+  HeaderSubtitle,
+  ButtonsWrapper,
+  HeaderButton,
+  LocationCopyWrapper,
+  HeaderSubCopyWrapper,
+  LastUpdatedDate,
+  RiskLevelThermometer,
+  RiskLevelWrapper,
+  RiskLevelTitle,
+  RiskLevel,
+  RiskLevelGraphicMobile,
+  RiskLevelGraphicDesktop,
+  Triangle,
+} from 'components/LocationPage/NewLocationPageHeader.style';
+import { useEmbed } from 'common/utils/hooks';
+import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
+import { Level } from 'common/level';
+import { COLOR_MAP } from 'common/colors';
+import { useModelLastUpdatedDate } from 'common/utils/model';
+import { STATES_WITH_DATA_OVERRIDES } from 'common/metrics/hospitalizations';
+import { Projections } from 'common/models/Projections';
+import { formatDate } from 'common/utils';
+import { isNull } from 'util';
+
+const RiskLevelGraphic = (props: {
+  projections: Projections;
+  alarmLevel: number;
+  levelName: string;
+}) => {
+  return (
+    <RiskLevelWrapper>
+      <RiskLevel>{props.levelName}</RiskLevel>
+      <RiskLevelThermometer alarmLevel={props.alarmLevel} />
+      <Triangle alarmLevel={props.alarmLevel} />
+      <RiskLevelTitle>Covid Risk Level</RiskLevelTitle>
+    </RiskLevelWrapper>
+  );
+};
+
+const LocationPageHeading = (props: { projections: Projections }) => {
+  const { isEmbed } = useEmbed();
+
+  const displayName = props.projections.countyName ? (
+    <>
+      <strong>{props.projections.countyName}, </strong>
+      <a
+        href={`${
+          isEmbed ? '/embed' : ''
+        }/us/${props.projections.stateCode.toLowerCase()}`}
+      >
+        {props.projections.stateCode}
+      </a>
+    </>
+  ) : (
+    <strong>{props.projections.stateName}</strong>
+  );
+
+  return <span>{displayName}</span>;
+};
+
+const noop = () => {};
+
+const NewLocationPageHeader = (props: {
+  projections: Projections;
+  condensed?: Boolean;
+  stats: { [key: string]: number | null };
+  onRtRangeClick?: () => void;
+  onTestPositiveClick?: () => void;
+  onIcuUtilizationClick?: () => void;
+  onContactTracingClick?: () => void;
+  onHeaderShareClick: () => void;
+  onHeaderSignupClick: () => void;
+  isMobile?: Boolean;
+}) => {
+  const hasStats = !!Object.values(props.stats).filter(
+    (val: number | null) => !isNull(val),
+  ).length;
+
+  const headerTopMargin = !hasStats ? -268 : -330;
+  const headerBottomMargin = !hasStats ? 115 : 0;
+
```

Comment:
why are these so ... specifc?

Reply:
Hmm yeah, so the header is drawn up onto the colored banner, and the margins change imprecisely depending on if the header includes stats or is just text.

<img width="1223" alt="Screen Shot 2020-06-05 at 7 15 38 PM" src="https://user-images.githubusercontent.com/44076375/83929361-2012f100-a761-11ea-8888-e5866ef16c00.png">

<img width="1220" alt="Screen Shot 2020-06-05 at 7 15 54 PM" src="https://user-images.githubusercontent.com/44076375/83929362-23a67800-a761-11ea-8fe1-2be1d61acf78.png">